docs/source/install.rst

```diff
@@ -6,11 +6,11 @@ You can install dask with ``conda``, with ``pip``, or by installing from source.
 Conda
 -----
 
-To install the latest version of Dask from the
-`conda-forge <https://conda-forge.github.io/>`_ repository using
+Binary Dask packages are maintained on both the ``defaults`` and `conda-forge
+<https://conda-forge.github.io/>`_ channels.  You can install them using
 `conda <https://www.continuum.io/downloads>`_::
```

Comment:
I don't know if you want detail here, which could easily confuse... Some points:
- dask is installed by default in anaconda, but not necessarily the latest version, they they don't need to install anything
- `defaults` is a channel alias, I don't think you can do `-c default`
- we no longer say how to specify conda-forge, but maybe having the link is enough
- conda is almost never seen on its own, so the link maybe should go to miniconda?